test/channel.test.ts

```diff
@@ -16,6 +16,33 @@ describe("Channel", () => {
   });
   $afterEach(() => scope.close());
 
+  describe("when we serially subscribe to a channel", () => {
+    $it("should not use the same event twice", async () => {
```

Comment:
Let's just put this in the main test body without using the global `it`. You should just be able to say:

```ts
it("does not use the same event twice when serially subscribed to a channel", function*() {
  // testcase here, automatically bootstrapped into Effection.
});
```